identity-apps-core/components/org.wso2.identity.apps.common/src/test/java/org/wso2/identity/apps/common/listener/AppPortalRoleManagementListenerTest.java

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright (c) 2023-2024, WSO2 LLC. (http://www.wso2.com).
```

Comment:
Change the year range in updated pom files as well

Reply:
This is now added for the two pom.xml files updated.